app/uk/gov/hmrc/transitmovementsrouter/controllers/errors/ConvertError.scala

```diff
@@ -79,4 +80,14 @@ trait ConvertError {
       case Unexpected(thr: Option[Throwable]) => PresentationError.internalServiceError(cause = thr)
     }
   }
+
+  implicit val objectStoreErrorConverter = new Converter[ObjectStoreError] {
```

Comment:
Please include test cases for this change in ``` ConvertErrorSpec ```

Reply:
Okay

Reply:
Done